web/chat/chat-tabs.react.js

```diff
@@ -1,18 +1,28 @@
 // @flow
 
+import type { AppState } from '../redux-setup';
+
 import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'lib/utils/redux-utils';
+import { unreadBackgroundCount } from 'lib/selectors/thread-selectors';
 
 import ChatThreadTab from './chat-thread-tab.react';
 import css from './chat-tabs.css';
 import ChatThreadHome from './chat-thread-home.react';
 import ChatThreadBackground from './chat-thread-background.react';
 
-type Props = {||};
+type Props = {|
+  unreadBgrCount?: number,
```

Comment:
1. Let's call this `unreadBackgroundCount`
2. `key?: type` means that the property may or may not be specified, but in your case it is always specified, just might be `undefined` sometimes. Instead, let's use `key: ?type`.

The difference is that `key?: type` matches both `{ }` and `{ key: undefined }`, but `key: ?type` only matches the latter.